src/centrifuge.js

```diff
@@ -735,11 +755,17 @@ export class Centrifuge extends EventEmitter {
           if (resolveCtx.next) {
             resolveCtx.next();
           }
+          if (this._config.onRefreshAttempt) {
+            this._config.onRefreshAttempt(true, resolveCtx);
```

Comment:
we need to create custom new context here that will only include information we want to export - at moment `resolveCtx` can contain some internal fields.

Reply:
Updated